src/components/Profile/Bookmarks.js

```diff
@@ -0,0 +1,128 @@
+// react
+import React, { Component } from 'react';
+
+// 3rd party libraries
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+// styles
+import './Bookmarks.scss';
+
+// components
+import { formatDate } from 'utils';
+
+// actions
+import {
+  toggleBookmark,
+  fetchBookmarks,
+  removeBookmarkAction,
+} from 'store/actions/bookmarkActions';
+
+
+export class Bookmarks extends Component {
+  componentDidMount() {
+    const { dispatchFetchBookmarks } = this.props;
+    dispatchFetchBookmarks();
+  }
+
+  remove = (e) => {
```

Comment:
Also, the function name "remove" is not so descriptive of what it is doing. Perhaps you should consider maybe removeBookmark or anything in that realm!